net-im/discord-bin/discord-bin-0.0.10.ebuild

```diff
@@ -50,13 +50,7 @@ RDEPEND="
 S=${WORKDIR}
 
 QA_PREBUILT="
-	opt/discord/${MY_BIN}
-	opt/discord/libEGL.so
-	opt/discord/libGLESv2.so
-	opt/discord/swiftshader/libEGL.so
-	opt/discord/swiftshader/libGLESv2.so
-	opt/discord/libVkICD_mock_icd.so
-	opt/discord/libffmpeg.so
+	opt/discord/*
```

Comment:
There are 2 new binaries that needed to be included. Since this is a binary package it seems simpler/better to just mark everything as prebuilt, not much to gain from listing all the files separately I think.